draft-claise-semver.md

```diff
@@ -87,24 +86,23 @@ semver.org):
 * BETA is incremented when a new pre-release or testing version is made. 
 
 
-In IETF terms, this versioning scheme provides functionality equivalent to
+In IETF terms, this versioning scheme provides functionality analagous to
 several parts of the traditional IETF process.
 
-* MAJOR replaces the "obsoletes" relation between RFCs
-* MINOR replaces the "updates" relation between RFCs
-* PATCH replaces the IETF errata process
-* BETA replaces Internet-draft versions
+* MAJOR is analogous to an "obsoletes" relation between RFCs
+* MINOR is analagous to an "updates" relation between RFCs
+* PATCH is analgous to use of the IETF errata process
+* BETA is analgous to incrementing an Internet-draft version
 
 
 The more major a change to the specification, the more consensus is required.
-MAJOR changes MUST be subject to the IETF consensus process; every commit that
```

Comment:
I’m not sure I would have used normative language to describe the WG decision process, because people reading this will get wrapped around the axle about what is being REQUIRED when really we just want to have a shared understanding of how a WG could experiment with this if it chooses to do so. But wanted to chat about it before suggesting alternative non-normative phrasing.